packages/legacy/src/common/components/ProviderSelect.tsx

```diff
@@ -45,17 +48,28 @@ interface ITargetProviderSelectProps extends IProviderSelectBaseProps<IOpenShift
 
 type ProviderSelectProps = ISourceProviderSelectProps | ITargetProviderSelectProps;
 
+function toOptionWithValue (provider: IProviderObject) {
+  return {
+    toString: () => provider.metadata.name,
+    value: provider,
+  } as OptionWithValue<IProviderObject>;
+}
+
 export const ProviderSelect: React.FunctionComponent<ProviderSelectProps> = ({
   providerRole,
+  providerAllowRemote = true,
   field,
+  onProviderSelect,
   notReadyTooltipPosition = 'left',
   afterChange,
   namespace,
   ...props
 }: ProviderSelectProps) => {
   const inventoryProvidersQuery = useInventoryProvidersQuery();
   const clusterProvidersQuery = useClusterProvidersQuery(namespace);
+
   const { data: inventoryData } = inventoryProvidersQuery;
+  const { data: { items: providerItems = [] } = {} } = clusterProvidersQuery;
```

Comment:
usually I prefer destructuring but here (because it's a single property) I would vote for `?.` operator

Reply:
I thought for sure Radek would appreciate the destructure, but it does look cleaner the other way around.